docs/getting_started/creating_a_flyte_project.md

```diff
@@ -0,0 +1,79 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+---
+
+# Creating a Flyte project
```

Comment:
If you run the workflow in the `my_project` Flyte project in the demo cluster by running `pyflyte run --remote` (instead of packaging and registering the project), FlyteConsole will not show the name of your project—it says "Flytesnacks" instead, since that's the project name associated with the demo cluster. We should probably mention that in either this article or "Running a workflow locally", or both, to reduce confusion.

Reply:
`pyflyte run --remote -p <my project> -d development` should do the trick.. have you tried that?